homeassistant/components/huisbaasje/strings.json

```diff
@@ -0,0 +1,24 @@
+{
+  "title": "Huisbaasje",
+  "config": {
+    "step": {
+      "user": {
+        "title": "Huisbaasje",
```

Comment:
No need to specify title if it shouldn't be translated. The integration name will be used.
```suggestion
```